src/main/java/org/opentripplanner/routing/graphfinder/PatternAtStop.java

```diff
@@ -15,6 +17,8 @@
  *
  * TODO Is this the right package for this?
  */
+@ToString
+@EqualsAndHashCode
```

Comment:
Lombok can tak away a lot of boiler plate code, but we have not decided on using the framework. We used it to create the RoutingService (a fasade using the delegate annotation).

OTP uses the Object#equals for most classes that are part of the graph. OTP also have its own ToStingBuilders(one for entities and one for value objects).

Reply:
Lombok annotations have been replaced with `ToStringBuilder` and `Objects.equals/hash`.